### PR TITLE
ledger: don't require a URL with a path in it

### PR DESCRIPTION
**Pull request by andreimatei**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/examples-go/76)

### CODE REVIEW

**Comment by tamird** [comment]
There's some precedent for this in the other examples? LGTM

File: ledger/main.go [hunk 1]
```diff
@@ -243,6 +243,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	parsedURL.Path = "ledger"
```
Comment (tschottdorf) [inline ledger/main.go]: Shouldn't overwrite a previous `Path` though.

Comment (andreimatei) [inline ledger/main.go]: done

**Comment by andreimatei** [comment]
there's precedent, I copied it from another one

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/examples-go/76#-:-KRivi_Xgf2YHwmOy3bD:bt8t66m)*

**Pull request merged**